app/controllers/versions_controller.rb

```diff
@@ -1,13 +1,25 @@
 class VersionsController < ApplicationController
   before_action :find_rubygem
+  before_action :latest_version, only: [:show, :reverse_dependencies]
+  before_action :set_page, only: [:reverse_dependencies]
 
   def index
     @versions = @rubygem.versions.by_position
   end
 
   def show
-    @latest_version = Version.find_from_slug!(@rubygem.id, params[:id])
     @versions = @rubygem.public_versions_with_extra_version(@latest_version)
```

Comment:
if you're going to lazy load `latest_version` probably want to remove the `@` from this line